backend/pkg/sqlmanager/mysql/mysql-manager.go

```diff
@@ -34,7 +34,8 @@ func (m *MysqlManager) GetDatabaseSchema(ctx context.Context) ([]*sqlmanager_sha
 	for _, row := range dbSchemas {
 		var generatedType *string
 		if row.Extra.Valid && row.Extra.String == "GENERATED" {
-			generatedType = &row.Extra.String
+			generatedTypeCopy := row.Extra.String
+			generatedType = &generatedTypeCopy
```

Comment:
curious why this and the other examples needed to be changed here?

Reply:
linter was throwing an error due to an implicit memory aliasing - we were referencing a loop variable inside the for loop which can lead to all of the pointers pointing to the last item's data. 

````
pkg/sqlmanager/mysql/mysql-manager.go:37:20: G601: Implicit memory aliasing in for loop. (gosec)
                        generatedType = &row.Extra.String
```

Was doing a little [bit of reading](https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop) and it mentioned that g0 1.22 solves this but golang-ci lint might be using an older version so it still throws the error. 

I guess we could just ignore it if we're confident that it's not a real warning. Happy to revert if that makes sense.

Reply:
all good, i'd trust the linter in this case! Might be fine, but could also be a hidden bug too.